src/norm/sqlite.nim

```diff
@@ -95,7 +95,9 @@ proc createTables*[T: Model](dbConn; obj: T) =
 
   let qry = "CREATE TABLE IF NOT EXISTS $#($#)" % [T.table, (colGroups & fkGroups).join(", ")]
 
-  debug qry
+  when defined(normDebug):
+    debug qry
+
```

Comment:
@Clonkk you haven't put an blank line between debug and exec in other cases, so, for consistency, I'm asking you to remove it here.